Feat(YARA-621): Add match filter to systems table

adonispuente opened the pull request:
Ticket: https://issues.redhat.com/browse/YARA-621 
This adds the match filter from signature table and makes it default on page load

mkholjuraev commented on src/Components/SysTable/Helpers.js:
```diff
@@ -0,0 +1,44 @@
+            {
+                category: 'Status',
+                id: 'Status',
+                chips: buildChips(filters, 'Status')
```
It seems you do not have to pass function parameters to buildChips.

mkholjuraev commented on src/Components/SysTable/SysTable.js:
```diff
@@ -89,11 +100,10 @@ const SysTable = () => {
             }}
             noSystemsTable={<MessageState className='pf-c-card' icon={SearchIcon} variant='large' title={intl.formatMessage(messages.noResults)}
                 text={intl.formatMessage(messages.noResultsMatch)}/>}
-            customFilters={{
```
I believe that you can pass the value of match filter into custom params and inventory itself fires onRefreshData as soon as the filter value changes. This also enables not to have useEffect to fire onRefreshData from malware app (line 38)

adonispuente commented:
@mkholjuraev thanks for taking a look, I've added your changes, they definitely make it better. 
I updated the query so that if the argument is passed all results have the filter, if applied as opposed to the single page. I Also found/fixed a bug where deleting another filter would also result in deleting the match filter.
